app/assets/stylesheets/application.scss

```diff
@@ -277,6 +276,18 @@
     width: 60px;
   }
 
+//panel elements
+  .field-log-in {
+    color: #ead61c;
```

Comment:
Color literals like `#ead61c` should only be used in variable declarations; they should be referred to via variable everywhere else.